create_infotexts.py

```diff
@@ -461,6 +493,16 @@ def process_depicted_place(self, place_string, places_mapping, desc_string):
 
 
         self.data["depicted_place"] = place_as_wikitext
+        
+    
+    def process_keywords(self, item, mapping):
```

Comment:
An empty function!

Was the plan to have L237-242 inside this function? If so it should be a function under the CypernImage class instead.